### PR TITLE
Some performance improvement and refactoring

### PR DESCRIPTION
**Pull request by RAX7**
This PR contains a:

- [ ] **bugfix**
- [ ] new **feature**
- [x] **code refactor**
- [ ] **test update** 
- [ ] **typo fix**
- [ ] **metadata update**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -348,28 +348,28 @@ export function parseSrcset(input) {
   }
 }
 
-export function parseSrc(input) {
```
Comment (alexander-akait) [inline src/utils.js:351]: It was by design to reuse in some packages, so we should export it

File: src/utils.js [hunk 2]
```diff
@@ -815,7 +842,16 @@ function metaContentType(options) {
...
+//   try {
+//     source = trimC0ControlCodes(source);
+//   } catch (error) {
+//     throw new HtmlSourceError(
+//       `Bad value for attribute "${options.attribute}" on element "${options.tag}": ${error.message}`,
+//       options.attributeStartOffset,
+//       options.attributeEndOffset,
+//       options.html
+//     );
+//   }
```
Comment (alexander-akait) [inline src/utils.js:854]: Why it commented?

**Review by alexander-akait — COMMENTED** [review]
Thank you for PR, can you provide changes in perf?

File: src/utils.js (hunk 1 above)
Comment (RAX7) [reply src/utils.js:351]: Ok, I'm rolling it back

File: src/utils.js (hunk 2 above)
Comment (alexander-akait) [reply src/utils.js:854]: Strange :smile:

**Comment by RAX7** [comment]
@alexander-akait benchmark test: https://jsben.ch/y89Fx

**Review by alexander-akait — DISMISSED** [review]

**Comment by alexander-akait** [comment]
@RAX7 Looks like some tests failed, can you check it?

**Comment by alexander-akait** [comment]
Thank you

**Pull request merged**